tests/test-buttercup.el

```diff
@@ -15,6 +15,9 @@
 ;; You should have received a copy of the GNU General Public License
 ;; along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+;;; Commentary:
+;;
+
```

Comment:
Do me this is an example of a case where I don't care about the empty section but I care about the absence of warnings in my file. I added some comments.